src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,48 @@
+package duke.task;
+
+/**
+ * Represents a task
+ */
+public class Task {
+
+    public static final String DONE_MARKER = "X";
+    public static final String NOT_DONE_MARKER = " ";
+
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Task constructor.
+     *
+     * @param description Description of task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns description of task.
+     *
+     * @return Description of task.
+     */
+    public String getDescription() {
+        return this.description;
+    }
+
+    /** Sets isDone to given boolean */
+    public void setIsDone(boolean b) {
+        isDone = b;
+    }
```

Comment:
Could this be better if it just marks the task as done instead of having to pass in a boolean parameter?
```suggestion
    public void setDone() {
        isDone = true;
    }
```